File: qe-selenium-java/src/main/java/demos/ActivityOne.java
package demos;

import org.openqa.selenium.firefox.*;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class ActivityOne {
    public static void main(String[] args) throws InterruptedException {
        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net");

        System.out.println(driver.getTitle());

        driver.findElement(By.linkText("About Us")).click();
        Thread.sleep(200);

        System.out.println(driver.getTitle());

        driver.quit();
    }

}

File: qe-selenium-python/activityOne.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from time import sleep

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net")

    print(driver.title)

    driver.find_element(By.LINK_TEXT, "About Us").click()
    sleep(2)

    print(driver.title)


    driver.quit()


